### ISSUE TITLE
[BUG] incorrect environment variables being selected/used

### ISSUE THREAD

**Issue by mrtristan** [getinsomnia/insomnia#796]
insomnia: 5.14.9
windows 10

![image](https://user-images.githubusercontent.com/1546181/36928609-4f6568bc-1e55-11e8-9d85-7ea8c10a3dd5.png)

I'm currently set to the QA env. if i hover over the ApiPath variable, it shows the local env value. if i hover over the email / password variables, it shows the QA values. it also USES the incorrect values.

local env:

```
{
	"APIPath": "localhost",
	"Email": "some local email",
	"Password": "some local password"
}
```

qa env:

```
{
	"APIPath": "http://qa-egapi.xxxx.com",
	"Email": "some qa email",
	"Password": "some qa password"
}
```

**Comment by mrtristan** [comment]
deleting the "local" env still yields the local env value going in to that variable

**Comment by mrtristan** [comment]
deleting and re-adding the variable still yields wrong value

**Comment by gschier** [comment]
Do you have anything in your base environment?

**Comment by gschier** [comment]
Or in a folder environment?

**Comment by mrtristan** [comment]
i did not realize folder environments was a thing until your question just now. i checked each folder and none of them have anything specified

**Comment by gschier** [comment]
Hmm, this is very strange. Does it happen even after quitting and starting the app?

If so, can you try duplicating the workspace (from workspace settings) and let me know if it still happens in the new workspace?

**Comment by mrtristan** [comment]
yep. happens after quit / relaunch. also happens in a duplicated workspace. if you'd like to externally connect to do a deeper dive in to whats happening just let me know.

**Comment by gschier** [comment]
Sure. Do you want to hop on the [Slack](https://chat.insomnia.rest/) group? Should be easy to track down the problem if we can discuss in real-time.

**Comment by mrtristan** [comment]
i found that there was an override of a single property that was common across my environments at a top level folder
